definitions/npm/moment-timezone_v2.x.x/flow_v0.104.x-/moment-timezone_v2.x.x.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Flowtype definitions for moment-timezone
+ * Generated by Flowgen from a Typescript Definition
+ * Flowgen v1.10.0
+ */
+
+import * as moment from "moment";
+declare module.exports: typeof moment;
```

Comment:
Those two lines can't work with flow-typed.

Reply:
If you need types from "moment" you'll need to copy them here. You won't be able to import them.

Reply:
`moment-time` is not an independent library. it is based on `moment`, the usage will like `moment.tz()` or `moment('times-str').tz('TIMEZONE', boolean?)`

Reply:
Please see https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#dont-import-types-from-other-libdefs